### PR TITLE
[WIP] Release candidate

### PR DESCRIPTION
**Pull request by ianegordon**
DO NOT MERGE

### CODE REVIEW

**Comment by ianegordon** [comment]
Commit authors have be verified.

**Comment by romoore** [comment]
Commit 58761c7 would indicate that our "supplemental" approach is a bit risky.

**Comment by romoore** [comment]
Why did we have to revert the `shadowPath` work? I recall it was pretty critical for a client team.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,142 @@
+# 60.1.0
+
```
Comment (jverkoey) [inline CHANGELOG.md:2]: There's no description of this release?

Comment (romoore) [reply CHANGELOG.md:2]: Do you mean naming it "v60.1.0"?  I don't think we normally name our releases, do we?

Comment (jverkoey) [reply CHANGELOG.md:2]: https://github.com/material-components/material-components-ios/blob/develop/contributing/releasing.md#new-features--top-level-description

The release engineer is expected to summarize the release’s contents at the top of the release notes, notably calling out any new features that may have been added.

**Pull request merged**